project-system-tools/.github/workflows/teams-notifications.yml

```diff
@@ -0,0 +1,18 @@
+name: Post PRs and issues to Teams
```

Comment:
We don't want two of these; we already have a notifcation that notifies teams. Just remove this.